### DISCUSSION TITLE
User Ops dissapear after Cloning Patch/reloading Patch

### DISCUSSION THREAD

**Discussion by RappScallion1** [cables-gl/cables_docs#526] [Answers & Questions]
Hello all,

I have stumbled across a problem with the userOps.
I'm trying to create a custom color library and for that I cloned the Ops.Array.PaletteLibrary.

![UserOP_1](https://user-images.githubusercontent.com/131858813/234521673-4d7a1272-0bfc-41b3-ab26-ae5394204c34.png)
![UserOP_2](https://user-images.githubusercontent.com/131858813/234521683-20d8ad61-606a-4df9-9ef3-ae18e616e603.png)

Then I add my own colors and link my userOp in my patch.

![UserOP_3](https://user-images.githubusercontent.com/131858813/234521685-ceaf0138-5276-4346-8ab5-0ac35bf03045.png)

![UserOP_4](https://user-images.githubusercontent.com/131858813/234521686-5ae026d4-794b-4591-921b-517ac14c9392.png)

So far everything works, but when I clone or reload the patch my newly created Op disappears.

![UserOP_5](https://user-images.githubusercontent.com/131858813/234521689-366e1dc9-ee5e-4748-8ed8-01608907abbc.png)
![UserOP_6](https://user-images.githubusercontent.com/131858813/234521692-0161d7db-9fe4-44b1-a7d9-eca7b42a4627.png)

I don't know exactly what the problem is, if it's a bug or if I missed a step.

![UserOP_7](https://user-images.githubusercontent.com/131858813/234521693-38944fdd-26c9-47dc-98ed-553589f2355f.png)

Does anyone else have the problem with the userOps or is it just me? 

Thanks, Rapp Scallion

**Comment by steam0r** [comment]
is this reproducible? i guess this is on cables.gl and not dev.cables.gl? are you sure you saved both the op-editor and the patch?
can you add me ("stephan") as a collaborator on a patch so i can have a look? thanks!
